superset/assets/javascripts/chart/ChartBody.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import $ from 'jquery';
+
+const propTypes = {
+  containerId: PropTypes.string,
+  vizType: PropTypes.string,
+  height: PropTypes.func,
```

Comment:
I'd make these required or add defaults, if they're not passed now your `this.props.width` etc calls below would throw